File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Description.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Description{

	@SerializedName("simpleText")
	private String simpleText;

	public void setSimpleText(String simpleText){
		this.simpleText = simpleText;
	}

	public String getSimpleText(){
		return simpleText;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Description.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class Description : Parcelable, Serializable {
    @SerializedName("simpleText")
    var simpleText: String? = null
    override fun toString(): String {
        return "Description{" +
                "simpleText = '" + simpleText + '\'' +
                "}"
    }
}